sql/original_update_tests.md

```diff
@@ -0,0 +1,48 @@
+# Original Update Tests
+This file is a port of the original hardcoded update-tester tests to markdown, similar to how sql-doctester works.
+
+These markdown files are parsed in the same way as sql-doctester would with the addition of two fields: `creation` and `validation`. (Names are not final).
+
+Tests marked `creation` will be ran during the object creation step, tests marked validation will be ran during the validation step.
+ 
+
+ 
+## Object Creation Step
+ 
+Currently, `SET TIME ZONE 'UTC';` will cause tests to fail if used in the `validation` step, so it should used in `creation` if needed.
+```sql,creation
+SET TIME ZONE 'UTC';
+
+CREATE TABLE test_data(ts timestamptz, val DOUBLE PRECISION);
```

Comment:
Normally, tests are fully isolated from each other.  But our very goal here is persistence.

Is it feasible to create a separate database for each test?  Or is that too expensive?

If it's too expensive, how can we prevent tests from stepping on each other?  Can we require that each test creates only tables prefixed with the test's name?  So, update-tester would require after running this file that only tables whose names begin with 'original_update_tests' were created.

This might be a bit of work, so we might want to consider this for a follow-up rather than doing it here.

Reply:
Sql-doctester seems to run fine and I believe that it also creates a separate database for each test file. We can prevent tests from stepping on each other by porting over the transactional/non-transactional functions from sql-doctester which would allow us to rollback any changes after a test runs.
I'm also ok with the table prefix idea.

Reply:
Updated release automation plan to include this in future work.